ci-operator/config/openshift/cluster-api-provider-metal3/openshift-cluster-api-provider-metal3-main.yaml

```diff
@@ -0,0 +1,41 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.20
+promotion:
+  name: "4.14"
+  namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.14"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.14"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+test_binary_build_commands: make unit
+tests:
+- as: e2e-aws
```

Comment:
this should probably not be here, but it's ok, we can change the tests later as far as we're good with the rest